### ISSUE TITLE
option --no-colors is not interpreted anymore - regression with MR 2187

### ISSUE THREAD

**Issue by Benoit12345** [mikefarah/yq#2218]
**Describe the bug**
Since the version 4.44.4/4.44.5, the [MR 2187](https://github.com/mikefarah/yq/pull/2187) introduces an issue: the option "--no-colors" or "-M" is not used anymore.

Version of yq: 4.44.4/4.44.5
Operating system: linux
Installed via: docker

**Input Yaml**
any  valid yaml

**Command**
The command you ran **with issue** (legacy command -> only '--no-colors' option):
```
docker run --rm -it -v /home/api:/api -w /api --security-opt=no-new-privileges --cap-drop all --network none mikefarah/yq:4.44.5 eval-all --no-colors '. as $item ireduce ({}; . * $item)' ./swagger1.yaml ./swagger2.yaml
```

The command you ran **without issue** (NO_COLOR is set + '--no-colors' option) :
```
docker run --rm -it -v /home/api:/api -w /api -e NO_COLOR=1 --security-opt=no-new-privileges --cap-drop all --network none mikefarah/yq:4.44.5 eval-all --no-colors '. as $item ireduce ({}; . * $item)' ./swagger1.yaml ./swagger2.yaml
```

**Actual behavior**

If there's only the '--no-colors' option, the output **has colors**

**Expected behavior**

If there's only the '--no-colors' option, the output **mustn't have colors** (legacy behavior -> compatibility)

**Comment by mikefarah** [comment]
doh! Thanks for raising, will fix

**Comment by mikefarah** [comment]
Fixed in 4.4.6
